doc/001-overview.md

```diff
@@ -49,42 +49,42 @@ Below is an outline of all the applications that are generated from this reposit
 
-| Binary Name       | Description|
+| Command           | Description|
 |-------------------| -----------|
-| ziti-controller   | Runs a central server necessary for Ziti environments|
-| ziti-router       | Runs a server capable of ingress'ing and egress'ing Ziti traffic standalone or as a mesh|
+| ziti controller   | Runs a central server necessary for Ziti environments|
+| ziti router       | Runs a server capable of ingress'ing and egress'ing Ziti traffic standalone or as a mesh|
 
 ### Tools
 
-The following binaries provide utility or testing functionality.
+The following commands provide utility or testing functionality.
 
-| Binary Name       | Description|
+| Command           | Description|
 |-------------------| -----------|
-| ziti-enroller     | Provides enrollment processing features for executables that do not directly support enrollment
-| ziti-fabric-gw    | Provides JSON RCP web service access to Ziti fabric management features
-| ziti-fabric-test  | The Ziti Fabric Toolbox which is used to test deployed fabric components|
+| ziti edge enroll  | Provides enrollment processing features for executables that do not directly support enrollment|
+| ziti fabric inspect | Observe Ziti Fabric runtime values                       |
 
 ### Management
 
 The following binaries are used to configure and manage a Ziti environment via command line interactions.
 
-| Binary Name       | Description|
+| Command           | Description|
 |-------------------| -----------|
-| ziti-fabric       | Provides command line access to Ziti Fabric management features|
-| ziti              | Provides command line access to Ziti management features|
+| ziti fabric       | Provides command line access to Ziti Fabric management features|
+| ziti edge         | Provides command line access to Ziti management features|
 
 ## Endpoint Clients
 
 The following binaries are Ziti endpoint clients which have the Ziti SDK built into them and can connected to an
 edge router. Endpoint clients can be application specific or act as a bridge to other applications, hosts, or underlay
 networks.
 
-| Binary Name       | Description|
+| Command           | Description|
 |-------------------| -----------|
-| ziti-tunnel       | Provides the ability to intercept traffic to route traffic across Ziti|
+| ziti tunnel       | Provides the ability to intercept traffic to route traffic across Ziti|
 
-All of the above binaries are cross platform compatible, except ziti-tunnel which is currently Linux only.
+All of the above binaries are cross platform compatible, except `ziti tunnel` which is currently Linux-only.
 
 ## Further Exploration
 
-Continue your OpenZiti exploration in [the next article about getting set up for local development](./002-local-dev.md) and building the apps from source.
+Continue your OpenZiti exploration in [the next article about getting set up for local development](./002-local-dev.md)
+and building the apps from source.
```

Comment:
please avoid unnecessary line splitting. it jsut makes reviewing harder

Reply:
It's necessary to split these lines in the vicinity of 80-120 columns because this Markdown is intended for reading in a code editor by a developer who has cloned the repo. If the primary audience were web clients reading HTML then we should not split the lines, and instead optimize for reviewing as you suggest.